src/app/user-profile/pages/edit-profile-page/edit-profile-page.component.ts

```diff
@@ -58,13 +59,57 @@ export class EditProfilePageComponent implements OnInit {
       nonNullable: true,
       validators: [
         Validators.required,
-        passwordStrengthValidator(),
+        this.UserPasswordStrengthValidator(),
       ],
     }),
   });
 
+  public UserPasswordStrengthValidator(): ValidatorFn {
```

Comment:
what's the reason for creating new validator?

Reply:
deleted and implemented already created validator